src/mpcCoreKit.ts

```diff
@@ -196,8 +196,14 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     return this.options.uxMode === UX_MODE.REDIRECT;
   }
 
+  // use dkg is always false for ed25519 key type and for secp256k1 key type when options.useDKG is false
   private get useDKG(): boolean {
-    return this.keyType === KeyType.ed25519 && this.options.useDKG === undefined ? false : this.options.useDKG;
+    return this.keyType === KeyType.ed25519 ? false : this.options.useDKG;
```

Comment:
what's the reasoning for disabling dkg completely for ed25519? some customers might still want it? i would think having the option is beneficial?

Reply:
because there is no dkg running for ed25519.

Reply:
got it. but we still could have the differentiation between "user imports seed" (with export capabilities) and "user generates share, server generates share" (where key never exists in one place, even though server side share is not coming from DKG)?